src/components/Layout/Flex/Flex.tsx

```diff
@@ -1,17 +1,38 @@
 import React, { forwardRef } from 'react'
+import { composeClasses } from 'lib/classes'
+import { AlignItems, JustifyContent, Gap } from '../../../interfaces/types'
 
-interface FlexProps extends React.HTMLAttributes<HTMLDivElement> {
+export interface FlexProps extends React.HTMLAttributes<HTMLDivElement> {
+  gap?: Gap
+  justifyContent?: JustifyContent
+  alignItems?: AlignItems
   className?: string
   children?: React.ReactNode
 }
 
 const Flex = forwardRef<HTMLDivElement, FlexProps>(
-  ({ className, children, ...props }: FlexProps, ref) => {
+  (
+    {
+      className,
+      alignItems = 'center',
+      justifyContent = 'between',
+      gap,
```

Comment:
We could add the "flex Wrap" and "flex Direction" props, or add flexFlow which handles both